control/include/robocup.hpp

```diff
@@ -11,4 +11,6 @@
  */
 #define USING_RTOS
 
+#define NORMAL_BASESTATION_PORT

```

Comment:
so for the alt robot port, we just delete this define and flash? should probably add a comment here explaining what this is + how to do it

Reply:
done